.github/workflows/cd.yml

```diff
@@ -0,0 +1,59 @@
+name: CD
+
+# See article for setting up github site pages for docs;
+# https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+jobs:
+  publish:
+    name: Publish Fluvio Client to NPM
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node: [ '12', '14' ]
+    steps:
+      - uses: actions/checkout@master
+        with:
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
+          fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+      - name: Use Node.js ${{ matrix.node }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Publish
+        run: |
+           make run_publish
```

Comment:
this should not be trigger for every push but for release. see example of how to trigger on tags: https://github.com/actions/create-release

Reply:
commit 072af98 now runs workflow on pushing a tag;